commands/droplets.go

```diff
@@ -556,9 +556,35 @@ func matchDroplets(ids []string, ds do.DropletsService, fn matchDropletsFn) erro
 
 // RunDropletGet returns a droplet.
 func RunDropletGet(c *CmdConfig) error {
+
 	id, err := getDropletIDArg(c.NS, c.Args)
 	if err != nil {
```

Comment:
This check can be removed. It always results in an error when passing a string. `getDropletIDArg` ensures that only one argument is provided and that it is an integer. Now that integers and strings are valid arguments, the check you added below will be enough to validate input.